CONTRIBUTING.rst

```diff
@@ -48,6 +48,11 @@ It is also possible to automatically check if all links are still valid::
 
    python setup.py build_sphinx -b linkcheck
 
+If you have ``katex`` installed, you can also check the prerendered
+version of the documentation with::
+
+    SPHINXCONTRIB_KATEX_PRERENDER=1 python setup.py build_sphinx
```

Comment:
I removed this in #15 as there is a sphinx build in option to apply specific configurations:
```bash
$ python -m sphinx doc/ build/sphinx/ -b html -D katex_prerender=1
```